src/main/java/com/alibaba/fastjson/JSON.java

```diff
@@ -564,6 +564,24 @@ public static JSONArray parseArray(String text) {
         return list;
     }
 
+    public static boolean isJSON(String text) {
```

Comment:
not necessary,it's simple and enough to provide two method in your application codes.

Reply:
But I think at least one verification method should be provided . For example : 
`boolean validate(String text)`
:)